### PR TITLE
M65 bug fixes and updates

### PR DESCRIPTION
**Pull request by alinelena**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/m65/rev2/mcuconf.h [hunk 1]
```diff
@@ -0,0 +1 @@
+../rev1/mcuconf.h
```
Comment (fauxpark) [inline keyboards/m65/rev2/mcuconf.h:1]: Please delete this file, and the readme.md.

File: keyboards/m65/rev2/chconf.h [hunk 2]
```diff
@@ -0,0 +1 @@
+../rev1/chconf.h
```
Comment (fauxpark) [inline keyboards/m65/rev2/chconf.h:1]: Please delete this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/m65/rev2/mcuconf.h (hunk 1 above)
Comment (alinelena) [reply keyboards/m65/rev2/mcuconf.h:1]: done thank you, removed symlink added the file

File: keyboards/m65/rev2/chconf.h (hunk 2 above)
Comment (alinelena) [reply keyboards/m65/rev2/chconf.h:1]: removed symlink added the file

**Review by fauxpark — APPROVED** [review]

File: keyboards/m65/readme.md [hunk 3]
```diff
@@ -19,4 +22,20 @@ Flashing example for this keyboard:
...
+
+    make m65/rev2:default:flash
+
+To enter flashing mode, on microcontroller keep pressed KEY and then NRST, release NRST and then after 1s release KEY.
+
+For via any rev
+
+    make m65/rev2:via
+
+    make m65/rev1:via
```
Comment (fauxpark) [inline keyboards/m65/readme.md:39]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by alinelena** [comment]
super seeded by https://github.com/qmk/qmk_firmware/pull/14381

**Pull request closed without merging**